content/docs/introduction/billing.md

```diff
@@ -0,0 +1,199 @@
+|**RAM**                   | 1 GB                              | Up to X GB       | Up to X GB                | Up to X GB              |
+|**Storage**               | 3 GB per branch                   | Up to X GB       | Up to X GB                | Up to X GB              |
+|**Dedicated resources**   | &#120;                            | &#120;           | &check;                   | &check;                 |
+|**Auto-suspend compute**  | &check;                           | &check;          | &check;                   | &check;                 |
+|**Always-on compute**     | &#120;                            | &check;          | &check;                   | &check;                 |
+|**Project sharing**       | &#120;                            | &check;          | &check;                   | &check;                 |
+|**Payment**               | Free                              | Credit Card, Pay As You Go with monthly invoicing | Prepaid, Custom Contract, Volume Discounts |Prepaid, Custom Contract, Wholesale Discounts|
+|**Support**               | Community Support                 | Support          | Support                   |Support, Platform Support|
+
+<Admonition type="info">
+The limits described above are plan defaults. If you would like to adjust the limits to tailor a plan to your specific requirements, please contact [sales@neon.tech](mailto:sales@neon.tech).
+</Admonition>
+
+Our [Pricing](https://neon.tech/pricing) page provides additional information and a calculator you can use to estimate costs and determine which plan is right for you.
+
+## Account billing page
+
+Each Neon account has a billing page, where you can:
+
+- View your current billing totals
+- Start a paid subscription
+- Update your payment details
+- Download current and previous invoices
+
+To access your billing page:
+
+1. Navigate to the Neon Console.
+1. Select **Billing** from the sidebar.
+
+## Neon invoices
+
+A Neon invoice includes the total cost for the billing period and the cost broken down by [billing metric](#neon-billing-metrics).
+
+### Download invoices
+
+You can download invoices from the **Billing** page.
+
+1. Navigate to the **Billing** page in the Neon Console. The current invoice is displayed.
+1. Download the invoice:
+    1. To download the current invoice, click **Download PDF** from the top of the page.
+    1. To download an invoice for a previous billing period, select the invoice from the **Latest invoices** list to open it, then click **Download PDF**.
+
+## Cancel a subscription
+
+To cancel your subscription to a Neon paid plan:
+
+1. Navigate to the **Billing** page in the Neon Console.
+1. Click **Cancel subscription**.
+1. Enter your cancellation request and click **Submit**.
+
+This action initiates the cancellation. If your data exceeds  free-tier storage limits, you will be contacted by the Neon Support team with a request to reduce your storage before the paid plan is canceled and free-tier limits are applied.
```

Comment:
What will happen if the user won't reduce the storage to fit into limits?

Reply:
@acervantes23 What will happen if the user won't reduce the storage to fit into limits? Would we "archive" the project or would it be removed?

Reply:
@danieltprice from our Cancelation guide: 

We will determine which project seams to be the most active and suspend all other projects. In addition, we will determine which branches are the most active and suspend access to all other branches. 

 We will keep the projects and storage for up to 60 days with the option to remove this afterwards.

Reply:
Link to guide: https://www.notion.so/neondatabase/Cancellation-Policy-direct-sales-self-serve-6608fe65a66f4a679d1e300b10bfca54?pvs=4

Reply:
I think we need a separate doc/section on the cancellation policy. We can link to it from here. I have created a Docs task in GitHub to do that.